app/dl_control_api/Dockerfile

```diff
@@ -19,6 +19,7 @@ COPY /README.md /src/$APP_DIR/README.md
 
 # Installation
 WORKDIR /src/metapkg
+ENV SETUPTOOLS_USE_DISTUTILS=stdlib
```

Comment:
I didn't spend too much time investigating the problem and followed https://github.com/pypa/setuptools/issues/3278, so there may be a cleaner solution
```
AttributeError: install_layout. Did you mean: 'install_platlib'?
```